pandas/tests/categorical/test_algos.py

```diff
@@ -71,6 +71,22 @@ def test_isin_empty(empty):
     tm.assert_numpy_array_equal(expected, result)
 
 
+def test_contains():
```

Comment:
move to test_indexing, should be other checks already there, de-duplicate with those.

Reply:
Lookef at test_indexing, didn't find anything very similar. I moved it to test_operators, which is more similar, IMO. Is that ok?